app/Http/Controllers/HomePage/CustomerController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Http\Controllers\HomePage;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\CustomerRequest;
+use App\Services\Customer\CustomerServices;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Session;
+
+class CustomerController extends Controller
+{
+    private $customerServices;
+
+    public function __construct(CustomerServices $customerSevices)
+    {
+        $this->customerServices = $customerSevices;
+    }
+
+    //Login screen
+    public function index()
+    {
+        return view('homepage.login', [
+            'title' => 'Login to store'
+        ]);
+    }
+
+    //Check login
+    public function checkLogin(Request $request)
+    {
+        $this->validate($request, [
+            'email' => 'required|email',
+            'password' => 'required|min:8',
+        ]);
+        $login = Auth::guard('cus')->attempt([
+            'email' => $request->input('email'),
+            'password' => $request->input('password')
+        ], $request->input('remember'));
+        if ($login) {
+            session()->put('customer', Auth::guard('cus')->user());
+            return redirect()->route('homepage.index');
+        }
+        return redirect()->back();
+    }
+
+    //Logout
+    public function logout()
```

Comment:
why login, logout method existed in CustomerController?

Reply:
I will make corrections